spec/lib/embed/media_tag_spec.rb

```diff
@@ -147,8 +156,12 @@
     end
 
     describe '#previewable_element' do
-      before { stub_purl_response_with_fixture(purl) }
+      before do
+        stub_purl_response_with_fixture(purl)
+        allow(file).to receive(:video_height).with(no_args)
```

Comment:
Hmm, is `.with(no_args)` required if we're not trying to assert that the method is invoked explicitly w/o args?